docs/rfc/rfc-1.adoc

```diff
@@ -234,6 +234,22 @@ and blinding factor, and then request that TBTC gets minted ASAP. Checking the
 dApp, they can see that they should expect TBTC in their provided wallet
 address in 3 hours with no further interaction.
 
+==== Fraud Proof
+
+There are only two valid types of transactions for a sweep:
```

Comment:
Sorry if I ask something that was once already explained to me...
Are we talking here about types of Bitcoin transactions that, if included as an argument of `notifySweep` Ethereum transaction, will be considered as non-fraudulent and will result in update of the `unsweptBalance` and `balance`?
If so, how is it that the second mentioned type of the transaction (refund) can be considered as a "valid type of transaction for a sweep"?

Reply:
Ah, good catch. I'll adjust the verbiage! It should say something along the lines of "there are only two valid types of transactions for an unswept deposit"

Reply:
addressed in 3077079